addons/stock/models/stock_picking.py

```diff
@@ -1321,3 +1321,14 @@ def _attach_sign(self):
         )
         return True
 
+    def _get_without_quantities_error_message(self):
+        """ Returns the error message raised in validation if no quantities are reserved or done.
+        The purpose of this method is to be overridden in case we want to adapt this message.
+
+        :return: Translated error message
+        :rtype: str
+        """
+        return _(
+            'You cannot validate a transfer if no quantities are reserved nor done. '
+            'To force the transfer, switch in edit mode and encode the done quantities.'
+        )
```

Comment:
could you move this method just after _should_show_transfers(self) in this file?